packages/mdc-select/foundation.js

```diff
@@ -216,9 +253,14 @@ export default class MDCSelectFoundation extends MDCFoundation {
     this.adapter_.setMenuElStyle('transform-origin', `center ${itemOffsetTop}px`);
   }
 
-  close_() {
+  close_(removeFloatingLabelClass) {
```

Comment:
Nit: rename param to `shouldRemoveFloatingLabelClass`, more indicative of being a boolean